{
  try {
    this.extractor.getFloatValue(this.person);
    fail("Should have throw an exception");
  }
 catch (  final Exception e) {
  }
}
